### ISSUE TITLE
Cloud Storage Pub/Sub notification

### ISSUE THREAD

**Issue by sjortiz** [GoogleCloudPlatform/deploymentmanager-samples#497]
Hello
All

Can you provide an example about attaching a Pub/Sub Notification to a bucket using Deployment manager? At the moment I can create the pub sub topic and the bucket apart but I don't find any way to attach them.

I've searched everywhere and there isn't anything on this.

Thanks in advance.

**Comment by ocsig** [comment]
Please take a look at our latest PubSub examples under the [Cloud Foundation Toolkit templates](https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/tree/master/dm/templates/pubsub).

This template creates the topic and the target and attaches them.

**Comment by jldupont** [comment]
@ocsig I do not believe this answers the question of @sjortiz (nor mine); we are interested in Storage Notification API (see https://cloud.google.com/storage/docs/json_api/v1/notifications ). Any help appreciated.
